src/follow-focus.cpp

```diff
@@ -95,6 +108,13 @@ class wayfire_follow_focus : public wf::plugin_interface_t
         last_coords = coords;
         last_view   = view;
 
+        if (!focus_delay)
+        {
+            do_actions();
+
+            return;
+        }
+
         /* Restart the timeout */
         change_focus.set_timeout(focus_delay, [=] ()
```

Comment:
wouldn't this work with a timer of 0 ? it'll just do it on the next idle cycle ? is it a big loss ?

Reply:
From my tests, trying to setup a timer with 0 timeout caused crash. This probably should be fixed in the `set_timeout()` implementation to just call the function directly and exit on case of 0 timeout.

Reply:
then really letting it be 5ms is virtually the same isn't it ? no human will detect anything happening in less than 50 ms.